packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -353,6 +356,11 @@ class ReactDataGrid extends React.Component {
 
   onGridRowsUpdated = (cellKey, fromRow, toRow, updated, action, originRow) => {
     const { rowGetter, rowKey, onGridRowsUpdated } = this.props;
+    // Deprecated prop
+    // to be removed in next major release
+    if (isFunction(this.props.onRowUpdated)) {
```

Comment:
We should add a warning message

Reply:
I have added deprecation warnings to all soon to be deprecated props